server/migrations/2020-03-20-224603_create_checkin_table/up.sql

```diff
@@ -2,7 +2,7 @@
 
 CREATE TABLE IF NOT EXISTS checkins (
     id SERIAL PRIMARY KEY,
-    gps_lat POINT NOT NULL, 
+    gps POINT NOT NULL, 
```

Comment:
Could you try this after enabling the postGIS extension too? If not, I'll do it later.